bin/martinize2

```diff
@@ -375,6 +382,14 @@ def entry():
                      'compatible. The -elastic and -govs-include flags cannot '
                      'be used together.')
 
+    file_extension = args.inpath.suffix.upper()[1:]  # We do not keep the dot
+    if file_extension in ['GRO'] and args.modelidx is not None:
+        parser.error("GRO files don't know the concept of models.")
```

Comment:
In principle they do: GRO files may contain multiple frames. They are not numbered, though, so it won't be entirely clear what model selection by number means.